### PR TITLE
Update README content and add new showcase mockup

### PR DESCRIPTION
**Pull request by dandaandaaaaaan**
## Changes
This PR updates the README page and adds a new screenshot for the front page.

## Closes Issues
* #54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,23 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103-W17-1/tp/actions)
```
Comment (sharpstorm) [inline README.md:1]: URL needs to change from `https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg` to `https://github.com/AY2122S2-CS2103-W17-1/tp/workflows/Java%20CI/badge.svg`

File: README.md [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2122S2-CS2103-W17-1/tp)
```
Comment (sharpstorm) [inline README.md:2]: URL needs to change from `https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg` to `https://codecov.io/gh/AY2122S2-CS2103-W17-1/tp/branch/master/graph/badge.svg`

File: README.md [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+##ContacX
```
Comment (sharpstorm) [inline README.md:6]: The name is wrong here
```suggestion
##ContaX
```

**Review by sharpstorm — CHANGES_REQUESTED** [review]
I noticed a few issues with changes.

File: README.md (hunk 1 above)
Comment (dandaandaaaaaan) [reply README.md]: thanks for catching, resolved in upcoming commit

File: README.md (hunk 2 above)
Comment (dandaandaaaaaan) [reply README.md]: thanks for catching, resolved in upcoming commit

File: README.md (hunk 3 above)
Comment (dandaandaaaaaan) [reply README.md]: thanks for catching, resolved in upcoming commit

File: README.md [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
+  * Management of contacts
+  * Management of schedules
+
+* Features:
+  * Detailed onboarding
+  * Add/Delete/Edit Contacts
+  * Contact Tagging and Filtering
+  * Advanced command chaining/mass operations
+  * Add/Delete/Edit Appointments
+* For the detailed documentation of this project, see the **[ContacX Product Website](https://ay2122s2-cs2103-w17-1.github.io/tp/)**.
```
Comment (sharpstorm) [inline README.md:20]: Typo here, please fix

Comment (dandaandaaaaaan) [reply README.md:20]: LOL i keep naming it wrongly lmao thanks for the catch

**Review by sharpstorm — APPROVED** [review]
lgtm.

**Review by Wei-Xinn — APPROVED** [review]
lgtm!

**Pull request merged**